xml/System.Threading.Tasks/Task.xml

```diff
@@ -791,7 +786,16 @@
           <see langword="true" /> to attempt to marshal the continuation back to the original context captured; otherwise, <see langword="false" />.</param>
         <summary>Configures an awaiter used to await this <see cref="T:System.Threading.Tasks.Task" />.</summary>
         <returns>An object used to await this task.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When an asynchronous method awaits a <xref:System.Threading.Tasks.Task> directly, continuation usually occurs in the same thread that created the task, depending on the async context. This behavior can be costly in terms of performance and can result in a deadlock on the UI thread. To signal your intention for continuation, call <xref:System.Threading.Tasks.Task.ConfigureAwait(System.Boolean)?displayProperty=nameWithType>. For more information, see [ConfigureAwait FAQ](https://devblogs.microsoft.com/dotnet/configureawait-faq/).
```

Comment:
```suggestion
When an asynchronous method awaits a <xref:System.Threading.Tasks.Task> directly, continuation usually occurs in the same thread that created the task, depending on the async context. This behavior can be costly in terms of performance and can result in a deadlock on the UI thread. To avoid these problems, call `Task.ConfigureAwait(false)`. For more information, see [ConfigureAwait FAQ](https://devblogs.microsoft.com/dotnet/configureawait-faq/).
```